### ISSUE TITLE
Update the Hello World template

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/paper-land#42]
While we plan to change the behavior of Create New Program so that it does not default create one from the Hello World template, it would be nice to have a thoroughly documented template Program that details what each section of the Program is intended for and our intended conventions (Line 1, keywords, async, Board, then Projector code)

Once we have keywords implemented (#36), I can probably draft this.

**Comment by BLFiedler** [comment]
I made a pass here to make sure the default template had the desired header:

> // Hello World Template
> // Keywords: start, begin, new, hello world
> // =============================== //
> // Program Dependencies: N/A
> // Recommended Programs: General Template (templates)
> // Program Description: Example program with functioning Board and Projector code!

I just added some functioning board code (Test Sound Clip) and left the Hello World projector code. We should revisit this with a better example of what the board is capable of, but this is a good starting point.

Unassigning, but I'm not completely comfortable with the current state, so I'll leave this open to address after we have a few more templates in place that might better serve here. Labeling on-hold for now.
